docs/source/app-dev/bindings-x-lang/index.rst

```diff
@@ -0,0 +1,92 @@
+
+Building Create Command
+=======================
+
+Let's recall an **IOU** example from the :doc:`Quickstart guide </getting-started/quickstart>`, where `Iou` template is defined like this:
+
+.. literalinclude:: ./code-snippets/quickstart/template-root/daml/Iou.daml
+  :language: daml
+  :lines: 9-15
+
+Here is how to manually build :ref:`com.digitalasset.ledger.api.v1.CreateCommand` for the above contract template in Scala:
+
+.. literalinclude:: ./code-snippets/iou-no-codegen/application/src/main/scala/com/digitalasset/quickstart/iou/IouCommands.scala
+   :start-after: // <doc-ref:iou-no-codegen-create-command>
+   :end-before: // </doc-ref:iou-no-codegen-create-command>
+
+If you do not specify any of the above fields or type their names or values incorrectly, or do not order them exactly as they are in the DAML template, the above code will compile but fail at run-time because you did not structure your create command correctly. Codegen is expected to simplify the command construction by providing auto-generated utilities to help you construct commands.
+
+When you use :doc:`Scala codegen </app-dev/bindings-scala/index>` to generate contract classes, a similar contract instantiation would look:
+
+.. literalinclude:: ./code-snippets/quickstart-scala/application/src/main/scala/com/digitalasset/quickstart/iou/IouMain.scala
+   :start-after: // <doc-ref:iou-contract-instance>
+   :end-before: // </doc-ref:iou-contract-instance>
+
+Building Exercise Command
+=========================
+
+To build :ref:`com.digitalasset.ledger.api.v1.ExerciseCommand` for `Iou_Transfer`:
+
+.. literalinclude:: ./code-snippets/quickstart/template-root/daml/Iou.daml
+  :language: daml
+  :lines: 23, 52-55
+
+manually in Scala:
+
+.. literalinclude:: ./code-snippets/iou-no-codegen/application/src/main/scala/com/digitalasset/quickstart/iou/IouCommands.scala
+   :start-after: // <doc-ref:iou-no-codegen-exercise-command>
+   :end-before: // </doc-ref:iou-no-codegen-exercise-command>
+
+versus creating the same command using a value class generated by :doc:`Scala codegen </app-dev/bindings-scala/index>`:
+
+.. literalinclude:: ./code-snippets/quickstart-scala/application/src/main/scala/com/digitalasset/quickstart/iou/IouMain.scala
+   :start-after: // <doc-ref:iou-exercise-transfer-cmd>
+   :end-before: // </doc-ref:iou-exercise-transfer-cmd>
+
+Summary
+=======
+
+If you are creating custom bindings for the Digital Asset distributed ledger, you will need to:
+
+- generate **Ledger API** from the gRPC definitions
+
+- decide whether to write a codegen to generate contract classes or manually build commands for all contracts defined in your DAML model.
+
+The above examples should help you get started. Feel free to contact us on the DAML community `Slack <https://damldriven.slack.com/sso/saml/start>`_ or open a `GitHub issue <https://github.com/digital-asset/daml/issues>`_.
```

Comment:
Totally up to you, but it might be better to link to the support page instead (`/support/support`). That way, we have one centralised way of telling people how to ask for help.